index.html

```diff
@@ -14,22 +14,30 @@
 </head>
 
 <body>
-    <header class="flex">
-        <div class="nav-bar flex">
+    <header class="">
+        <div class="nav-bar">
             <div>
                 <div class="topnav">
                     <a class="logo" href="#">My Logo</a>
```

Comment:
- [ ] Please prevent users to scroll down the page and see the content. You can achieve that by setting     

```
#mobile-menu {
position: fixed;
height: 100vh;
}
body {
overflow: hidden
}
```